controller/attribute/util.py

```diff
@@ -0,0 +1,137 @@
+import docker
+import json
+import os
+import pytz
+import re
+from datetime import datetime
+from submodules.model.business_objects import attribute, record, project, tokenization
+from submodules.model.enums import DataTypes
+from submodules.s3 import controller as s3
+
+client = docker.from_env()
+image = os.getenv("AC_EXEC_ENV_IMAGE")
+exec_env_network = os.getenv("LF_NETWORK")
+
+
+def find_free_name(project_id: str) -> str:
+    attribute_items = attribute.get_all(project_id, state_filter=[])
+    reg_ex = "^attribute_([0-9]+)$"
+    counter_list = []
+    for item in attribute_items:
+        match = re.match(reg_ex, item.name)
+        if match:
+            counter_list.append(int(match.group(1)))
+
+    if counter_list:
+        counter = max(max(counter_list), len(attribute_items)) + 1
+    else:
+        counter = len(attribute_items) + 1
+
+    return "attribute_" + str(counter)
+
+
+def add_log_to_attribute_logs(
+    project_id: str, attribute_id: str, log: str, append_to_logs: bool = True
+) -> None:
+    attribute_item = attribute.get(project_id, attribute_id)
+    berlin_now = datetime.now(pytz.timezone("Europe/Berlin"))
```

Comment:
this is an issue with other environments, too. for our managed version, this works perfectly, but generally, users aren't necessarily in Berlin in the OS variant. We need to change this some time down the road, please open an issue :) (not required to fix this in this PR, btw)